srcpkgs/yazi/template

```diff
@@ -13,13 +13,14 @@ license="MIT"
 homepage="https://yazi-rs.github.io"
 changelog="https://github.com/sxyazi/yazi/releases"
 distfiles="https://github.com/sxyazi/yazi/archive/refs/tags/v${version}.tar.gz"
-checksum=6aec4ed553670a47b3e5b34c161bb4356b1ebfac084b7354cd26052a81f971a4
+checksum=fe2a458808334fe20eff1ab0145c78d684d8736c9715e4c51bce54038607dc4e
 
 export VERGEN_GIT_SHA="${version}_${revision}"
 # enable the generation of shell auto completions
 export YAZI_GEN_COMPLETIONS=true
 
 post_install() {
+	vbin target/${RUST_TARGET}/release/ya
```

Comment:
Maybe just install it all (plus `yazi`) in a do_install step to avoid an additional compilation (drop the `make_install_path` as well).